lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_version_helper.rb

```diff
@@ -15,12 +17,33 @@ module VersionHelper
 
         # Returns the public-facing version string.
         #
+        # @param [String] xcconfig_file The path for the .xcconfig file containing the public-facing version
+        #
         # @return [String] The public-facing version number, extracted from the VERSION_LONG entry of the xcconfig file.
         #         - If this version is a hotfix (more than 2 parts and 3rd part is non-zero), returns the "X.Y.Z" formatted string
         #         - Otherwise (not a hotfix / 3rd part of version is 0), returns "X.Y" formatted version number
         #
+        def self.get_xcconfig_public_version(xcconfig_file:)
+          version = read_long_version_from_config_file(xcconfig_file)
+
+          UI.user_error!(".xcconfig file doesn't have a version configured") if version.nil?
+
+          vp = get_version_parts(version)
+          return "#{vp[MAJOR_NUMBER]}.#{vp[MINOR_NUMBER]}" unless is_hotfix?(version)
+
+          "#{vp[MAJOR_NUMBER]}.#{vp[MINOR_NUMBER]}.#{vp[HOTFIX_NUMBER]}"
+        end
+
+        # Returns the public-facing version string.
+        #
+        # @return [String] The public-facing version number, extracted from the VERSION_LONG entry of the xcconfig file.
+        #         - If this version is a hotfix (more than 2 parts and 3rd part is non-zero), returns the "X.Y.Z" formatted string
+        #         - Otherwise (not a hotfix / 3rd part of version is 0), returns "X.Y" formatted version number
+        #
+        # @deprecated This method is going to be removed soon due to it's dependency on `ENV['PUBLIC_CONFIG_FILE']` via `get_build_version`.
+        #
         def self.get_public_version
-          version = get_build_version
+          version = get_build_version()
```

Comment:
I was wondering why Rubocop didn't flag this but looks like that's still in the `rubocop-todo.yml` for this repo: https://github.com/wordpress-mobile/release-toolkit/blob/eeb93a90a037c33550c691c2595f28a70b9b9f3a/.rubocop_todo.yml#L256-L260

Reply:
After rebasing, now the style errors are being flagged 🎉 but for some reason several times 🙃 

https://github.com/wordpress-mobile/release-toolkit/pull/489#pullrequestreview-1481655717
https://github.com/wordpress-mobile/release-toolkit/pull/489#pullrequestreview-1481655762
https://github.com/wordpress-mobile/release-toolkit/pull/489#pullrequestreview-1481655808
https://github.com/wordpress-mobile/release-toolkit/pull/489#pullrequestreview-1481655854